packages/frontend/src/pages/scaling-activity/props/getProps.ts

```diff
@@ -10,16 +9,24 @@ import { getPageMetadata } from './getPageMetadata'
 
 export function getProps(
   config: Config,
-  activityApiResponse: ActivityApiResponse,
+  pagesData: PagesData,
 ): Wrapped<ActivityPageProps> {
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
This is not a good practice and is not necessary here - we are invoking this function only if `activityApiResponse` is defined. I assume it was necessary, because `PagesData['activityApiResponse']` is optional. Simpler solution is to define `pagesData` here with only required properties that are necessary to use in this function.